sdk/src/wallet/core/operations/storage.rs

```diff
@@ -76,8 +76,9 @@ mod storage_stub {
             storage: &impl StorageAdapter<Error = crate::wallet::Error>,
         ) -> crate::wallet::Result<Option<Self>> {
             log::debug!("get_wallet_data");
-            if let Some(data) = storage.get::<WalletData>(WALLET_INDEXATION_KEY).await? {
-                log::debug!("get_wallet_data {data:?}");
+            if let Some(data) = storage.get::<serde_json::Value>(WALLET_INDEXATION_KEY).await? {
+                log::debug!("get_wallet_data {data}");
+                let data = serde_json::from_value::<WalletData>(data)?;
```

Comment:
Is that just for the debug or was there an actual bug ?

Reply:
mmmm. I forget. Probably it was because the output is cleaner in json form

Reply:
Fixed it

Reply:
Wen read commit message? :sob: 
`change debug log to also log data if it's wrong`